pendulum_analysis/topic_statistics_analysis/topic_statistics_plot.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# Copyright 2020 Carlos San Vicente
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Plot topic statistics metrics')
+    parser.add_argument('filename', metavar='filename')
+    parser.add_argument('-s', '--show', help='Show plots', action='store_true')
+    parser.add_argument('-t', '--topicname', metavar='topicname', default=None)
+    parser.add_argument('-m', '--metric', metavar='metric', default='message_period',
+                        help='message_period or message_age')
+    parser.add_argument('-o', '--outfile',
+                        help='Name of file to write plot output')
+    args = parser.parse_args()
+    filename = args.filename
+    topicname = args.topicname
+    show = args.show
+    metric = args.metric
+    outfile = filename + '_plot'
+    if args.outfile is not None:
+        outfile = args.outfile
+    else:
+        if topicname:
+            outfile = topicname + '_' + metric
+        else:
+            outfile = 'topic_statistics' + '_' + metric
```

Comment:
```suggestion
            outfile = f"{topic_statistics}_{metric}"
```

Reply:
This should be: `outfile = f"topic_statistics_{metric}"`